lib/MetaCPAN/Web/Model/API/Permission.pm

```diff
@@ -89,6 +89,7 @@ sub _permissions_to_notification {
     Future->done( {
         took         => $data->{took} || 0,
         notification => $type,
+        notifiable   => [ grep !$special{$_}, @perm_holders ],
```

Comment:
This shouldn't be a separate item.  The `notification` value should be changed to a hash with type, email, and module elements.

Reply:
This is done.